src/styles/blocks/product.scss

```diff
@@ -0,0 +1,70 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  color: $main-font-color;
+
+  &__image {
+    background-image: url(/src/images/imac.jpeg);
+    height: 134px;
+    width: 160px;
+    background-size: 100% 100%;
```

Comment:
The `background-size` property set to `100% 100%` might distort the image if its aspect ratio does not match the element's aspect ratio. If maintaining the aspect ratio is important, consider using `cover` or `contain` instead.